spec/spec_helper.rb

```diff
@@ -1,5 +1,5 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
-require 'shore/customers/client'
+require 'shore/customers'
```

Comment:
This is the most important of the commits, and it's a commit of it's own, the `customers.rb` file requires both `client.rb` and `version.rb` files
requiring `client.rb` only, won't require `version.rb` at all, and this is why the `client_spec` fails.